hyperspy/io.py

```diff
@@ -478,6 +478,9 @@ def save(filename, signal, overwrite=None, **kwds):
         ensure_directory(filename)
         if overwrite is None:
             overwrite = overwrite_method(filename)
+        if overwrite is False:
```

Comment:
bah, was still not thorough enough, as inputting `overwrite='whatever'` was the same as `overwrite=True`, so now starting at line 479: 
```python
        if not(overwrite is True or overwrite is False):
            overwrite = overwrite_method(filename)
        if overwrite is False and os.path.isfile(filename):
            _logger.warning(f"file {filename} exists, did not overwrite")
        else:
            writer.file_writer(filename, signal, **kwds)
            ...
```